support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -334,6 +337,28 @@ export function ThreeTierLanding({
 		}
 	}, [memoizedCurrentCountdownCampaign]);
 
+	useEffect(() => {
```

Comment:
what does this useEffect do?

Reply:
I can't see how this useEffect has any effect though, as `getHeadline` just returns a value

Reply:
Removed, thanks